### PR TITLE
[ZFE-42] Insurance options from DB

### PR DESCRIPTION
**Pull request by zakrzaq**

### CODE REVIEW

**Review by elmir-ismayilzada — APPROVED** [review]
👍

File: components/Checkout/Insurance.vue [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <template>
   <div>
     <p class="mb-7 text-2xl">Insurance</p>
+    <div v-if="isLoading">...</div>
```
Comment (WawrzenskiPatryk) [inline components/Checkout/Insurance.vue:4]: `v-loading`? 🤔

File: components/Base/RadioGroup.vue [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
+          :value="opt"
+        >
+          <BaseRadioOption
+            :option="opt"
+            :checked="checked"
+          />
+        </RadioGroupOption>
+      </div>
+    </RadioGroup>
+  </ClientOnly>
```
[the remark this answers is not in the document]
Comment (elmir-ismayilzada) [reply components/Base/RadioGroup.vue:25]: Yes, RadioGroup component normally works only with it. I've no idea what happens under the hood on that component 🤷‍♂️

[the remark this answers is not in the document]
Comment (WawrzenskiPatryk) [reply components/Base/RadioGroup.vue:25]: @zakrzaq you told me at one of our daily meetings that `ClientOnly` was not a solution to the problem, was there something else about it I didn't get? 🤔

[the remark this answers is not in the document]
Comment (elmir-ismayilzada) [reply components/Base/RadioGroup.vue:25]: There's not only ClientOnly, also fixing props helped. He didn't do that

**Review by WawrzenskiPatryk — APPROVED** [review]
👍

**Pull request merged**